src/BoundingBox.php

```diff
@@ -66,6 +66,7 @@ public function getBounds(): array
         return $this->bounds;
     }
 
+    #[\ReturnTypeWillChange]
```

Comment:
Sorry for the delay, but I just had a change to review this. I'm don't believe that the `ReturnTypeWillChange` attributes are necessary, as the `jsonSerialize()` methods already specify `array` return type hints, which are [covariant](https://wiki.php.net/rfc/covariant-returns-and-contravariant-parameters) with the `mixed` return type specified for the [`JsonSerializable::jsonSerialize()`](https://www.php.net/manual/en/jsonserializable.jsonserialize.php) method.

Did you actually encounter deprecation notices when using this library?

The test suite runs on PHP 8.1 in CI, and I was only able to produce a deprecation notice by _removing_ type hints from the library's `jsonSerialize()` methods. For instance, removing the  `array` return type from `BoundingBox::jsonSerialize()` and running its test produces the following output:

> PHP Deprecated:  Return type of GeoJson\BoundingBox::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/jmikola/workspace/jmikola/geojson/src/BoundingBox.php on line 69